sympy/functions/elementary/trigonometric.py

```diff
@@ -1298,6 +1320,10 @@ def _eval_rewrite_as_sqrt(self, arg, **kwargs):
             return None
         return y
 
+    def _eval_rewrite_as_hyper(self, arg, **kwargs):
+        from sympy.functions.special.hyper import hyper
+        return (8*arg)/((pi**2) - (4 * (arg**2)))*hyper((1, (1/2) - (arg/pi), (arg/pi) + (1/2)), ((3/2) - (arg/pi), (arg/pi) + (3/2)), 1)
```

Comment:
Please avoid redundant parentheses. This is already long enough without them. You should also not put spaces around `*`. And finally, a long line like this should be wrapped (ideally in a way that makes it easiest to read, e.g., split up the arguments to `hyper`.